index.html

```diff
@@ -1,269 +1,429 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
-    <link rel="stylesheet" href="./styles/style.css">
-    <link rel="stylesheet" href="./styles/reset.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <link rel="stylesheet" href="./styles/reset.css" />
     <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-DW1CGEB4Y2"></script>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-DW1CGEB4Y2"
+    ></script>
     <script>
       window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
 
-      gtag('config', 'G-DW1CGEB4Y2');
+      gtag("config", "G-DW1CGEB4Y2");
     </script>
     <title>Michael Oladele | Portfolio</title>
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <nav class="nav-bar">
-            <ul>
-                <li id="name"> <a href="#">Michael Oladele</a></li>
-                <li class="hamburger-icon"><a href="#"><i class="fas fa-bars"></i></a></li>
-                <li class="right-item"> <a href="#">Portfolio</a></li>
-                <li class="right-item"> <a href="#">About</a></li>
-                <li class="right-item"> <a href="#">Contact</a></li>
-                <li class="right-item"> <a href="#"><i class="fas fa-envelope-square"></i></a></li>
-            </ul>
-        </nav>
+      <nav class="nav-bar">
+        <ul>
+          <li id="name"><a href="#">Michael Oladele</a></li>
+          <li class="hamburger-icon">
+            <a href="#"><i class="fas fa-bars"></i></a>
+          </li>
+          <li class="right-item"><a href="#">Portfolio</a></li>
+          <li class="right-item"><a href="#">About</a></li>
+          <li class="right-item"><a href="#">Contact</a></li>
+          <li class="right-item">
+            <a href="#"><i class="fas fa-envelope-square"></i></a>
+          </li>
```

Comment:
The **typography** ( font-family, font-weight, font size, etc), **spacing** (paddings/ margins) and **alignment** of your _mobile menu_ do not match the provided [desgn template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). We actually want our design to as much as possible closely mimic the design of the provided template. Kindly ensure to improve these:
- [ ] Typography should closely follow the given template
- [ ] The spacing on the extreme left and right should be equal and proportional to the given template
- [ ] The hamburger and the close button should be at the same position, but only interchangeable

**Your mobile menu design:**
![image](https://user-images.githubusercontent.com/12745474/135941026-6ef4624a-f4b3-47f9-8411-050beeb70b9e.png)

**The original hero design:**
![image](https://user-images.githubusercontent.com/12745474/135941139-fc67b9de-6843-4104-aff9-17e07c299eb7.png)